migrations/m160509_211405_create_notification_table.php

```diff
@@ -0,0 +1,25 @@
+<?php
```

Comment:
Your migration should patch the existing table using $this->addColumn(...) instead of re-creating it